### ISSUE TITLE
Feature Request: Required FW display + some other features

### ISSUE THREAD

**Issue by TheRadziu** [blastrock/pkgj#41]
NPS now has an extra column (Required FW) with exact FW game needs. This might allow you to add some features like being able to hide all 3.61+ titles (for 3.60 users) and being able to view only playable titles up to 3.65 (for 3.65 enso users) or even "view only 3.61+ games". 

Also, That would be great if you could use FW values to display it somehow in pkgj since I'd like to drop `(3.61+!)[X.XX]` from Name column.

**Comment by blastrock** [comment]
I'm looking into this... I guess the plan is:
- I update pkgj so that it ignores extra columns
- you update NPS's files to add that column
- I add support for it in pkgj

How does that sound to you?

**Comment by TheRadziu** [comment]
But the columns are already on nps for 2 months now xD

**Comment by blastrock** [comment]
Oh my bad, I was working with an old version of the file for a long time! The fact that it didn't break was not intended :P 

I guess I can implement this then.

**Comment by TheRadziu** [comment]
yeah, before pushing it for everybody I did check if NPS browser & latest pkgj were cool with the changes and they did. They simply didn't initialize those values :)

btw now you can also add "Hide all 3.61+" and "Hide all 3.65+"

**Comment by blastrock** [comment]
I was thinking of just hiding titles that can't install on the running firmware, I don't think adding an option for the user would be very useful.
